src/main/java/com/smartling/connector/exacttarget/sdk/rest/api/EmailApi.java

```diff
@@ -19,6 +20,12 @@
     @RequestLine("GET /asset/v1/content/assets/{id}/channelviews/html")
     Preview getEmailPreview(@Param("id") String id);
 
+    @RequestLine("POST /guide/v1/emails/{legacyId}/preview?kind=html")
+    EmailPreview getPreviewForEmail(@Param("legacyId") String legacyId);
```

Comment:
Can we use this endpoint to get preview for legacy emails (not content builder)? Just want to remove endpoint above and get rid this mess in names: `getEmailPreview`; `getPreviewForEmail`